### ISSUE TITLE
[Direwolf20 1.19] Some Gobber rings and medallions do not work.

### ISSUE THREAD

**Issue by ch4lplec** [FTBTeam/FTB-Modpack-Issues#2756]
### Modpack

FTB Legend of the Eyes

### Modpack version

1.19.2

### Describe the bug

The medallion of water breathing does not work. You still drown. 

The ring of sunlight does not clear the rain. 

I haven't tested any further as I'm in survival and haven't built anything else. 

### Steps to reproduce

1. Acquire gobber. 
2. Craft Gobber rings and medallions.
3. They don't work. Drowned in the lake outside my house and can't make the rain go away.

### Expected behaviour

Swim with fishes in the sunshine. 

### Screenshots

N/A

### Additional information

Playing on version 1.4.0 of the modpack in Forge 43.2.4.

**Comment by westingham** [comment]
I can confirm that the Medallion of Water Breathing does not work, whether it's in your inventory or in your Ender Chest.

Having said that, the Medallion of Experience, Ring of Curing, Ring of Repair, and Ring of the Phoenix DO work.

**Comment by Saereth** [comment]
yeah there are several that dont work, this is a known issue and reported to the mod dev. We await a fix for them at some point
